### DISCUSSION TITLE
Regular snapshot and motion snapshot

### DISCUSSION THREAD

**Discussion by picotrain77** [tsightler/ring-mqtt#560] [Support Q&A]
Hi 

This may be a stupid question but I am going to ask anyway as I can’t work it out myself…

I have the 2 generic camera’s setup in HA one for live and one for event. I have set the live camera to show snapshot static image and have set the snapshots to ‘all’.

I am struggling a little however with the event camera as I really wanted this to show a static snapshot that represented the last event and can’t work out if this is even possible. Obviously if I set it to show snapshot it simply always shows the same snapshot as the live camera. If I don’t add a snapshot then the picture shown seems to cycle through the event in a slightly odd way.

Apologies in advance if I have missed something here, I have searched and tried everything I can think off but can’t quite achieve what I wanted and would really appreciate any ideas or feedback.

**Comment by tsightler** [comment]
There is only a single snapshot camera, so the still image will always be the latest snapshot from that camera.

> If I don’t add a snapshot then the picture shown seems to cycle through the event in a slightly odd way.

If you don't set a still image for the snapshot then the behavior is just the default HA behavior.  I honestly have no idea what HA does in this case.

I think you could achieve what you are looking for by using an automation to save the snapshot when a motion/ding event occurs and then use that local file as the still image for the event camera.
